practices/development.md

```diff
@@ -2,105 +2,90 @@
-- {guilabel}`Ready to Work` Tasks that are ready to be worked on. Roughly speaking, tasks higher on the list are of higher priority.
-- {guilabel}`In progress` A task that a team member is currently working towards. When you move a task to In progress, indicate that you are working on it (either by adding your username to the card, or “assigning yourself” in the issue/PR associated with it.
-- {guilabel}`Needs review` Tasks that have an implementation that require feedback from others. For example, reviews for pull requests.
-- {guilabel}`Done` Tasks that are complete! When you move a task here, make sure to update any relevant deliverables.
+## Encoding longer-term efforts
 
-There is one Activity Backlog defined [at this GitHub Projects Board](https://github.com/orgs/2i2c-org/projects/5?fullscreen=true).
+Longer-term projects are generally more complex and may be made up of many actions and deliverables to accomplish.
+There is no official way to track long-term projects within 2i2c, but there are a few patterns that may be useful to do so.
 
-(coordination:tasks)=
-## Tasks
+% TODO: We should define a more reliable process for tracking long-term projects
 
-Deliverables of sufficient complexity are broken down into _tasks_. Tasks should be completable by a single person in less than one day (generally, completable with a few hours of focused work at most). Make sure to attach a link between your deliverable and its tasks (e.g., by adding links to issues in the deliverable’s "tasks" checklist)
+### Tracking issues
 
-Tasks are generally encoded as checklist items in a deliverable, then added to the Activity Backlog when the deliverable is being worked on.
+The simplest way to track longer-term efforts is with a **Tracking Issue**.
+This is a GitHub Issue whose job is to keep track of many actions and deliverables over time that are needed to close the issue.
+They are generally encoded as **Task Lists** in the issue's top comment.
+Each item in the list tends to be a deliverable, and can be converted into its own GitHub Issue (e.g., to put on the [Activity Board](coordination:activity-board)) as-needed.
 
-Here’s an example of a deliverable with a few tasks:
+(coordination:project-backlog)=
+### Project Backlogs
 
-- **Deliverable:** Side notes in Jupyter Books. \
-  **User story**: _As a book author, I want to be able to create “aside” content that does not break up the narrative flow of my text._ \
-  **Tasks:**
-    - Create design doc for directive name and general API
-    - PR to prototype functionality and documentation
-    - Add tests and QA
-    - Final approval + merge
+For more complex efforts, it can be useful to create a Project Backlog.
+These are GitHub Projects boards that contain all of the deliverables that will complete a given project.
```

Comment:
So, you are proposing to have per-project boards, I think that is a nice idea as long as we do not have dozen of boards :wink:
I guess the "current" deliverable board is actually one encompassing project that eventually would be partitioned into smaller ones. And I guess the org board could be seen as one of those particular projects, right?

Reply:
Well - here I am not trying to set a team **policy**, but suggesting a pattern that we can use and replicate. That said, yes I am suggesting that we track ongoing bodies of work with boards, and use those boards to help prioritize major things to work on next within a given project. I think this is project-specific and those boards should be closed as they are completed. That said, some projects might prefer something like GitHub Milestones to boards, to me the important thing is that we have *some* method for prioritizing deliverables, keeping track of the "big picture" and planning things out.

Reply:
Sounds reasonable to me. Thanks for the additional clarification.